reframe/core/logging.py

```diff
@@ -343,6 +343,9 @@ def __init__(self, logger=None, check=None):
                 'check_perf_ref': None,
                 'check_perf_lower_thres': None,
                 'check_perf_upper_thres': None,
+                'check_perf_upper_thres': None,
```

Comment:
Why is the `check_perf_upper_thres` two times here?

Reply:
copy and paste issues... sorry! 😯